Extend NS and FV documentation for NIST training (#23201)
idaholab/moose: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fvm_overview.md

!---

## The MOOSE Functor System

!style! fontsize=70%

> - It is unifies the evaluation and of Functions, Variables, certain Materials on faces, elements, boundaries.

Reviewer: ```suggestion
- Unifies the evaluation of Functions, Variables, and certain Materials on faces, elements, quadrature points, etc.
```
